nixos/tests/redis.nix

```diff
@@ -1,14 +1,13 @@
-import ./make-test-python.nix ({ pkgs, ... }:
+import ./make-test-python.nix ({ pkgs, lib, ... }:
 {
   name = "redis";
-  meta = with pkgs.lib.maintainers; {
-    maintainers = [ flokli ];
-  };
+  meta.maintainers = [ lib.maintainers.flokli ];
 
   nodes = {
     machine =
-      { pkgs, lib, ... }: with lib;
-
+      { pkgs, lib, ... }: let
+        inherit (lib) listToAttrs nameValuePair;
+      in
```

Comment:
I suggest to either prefix the functions with ``lib.`` or move the with lib to the function.